save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/PlatformUtils.kt

```diff
@@ -0,0 +1,35 @@
+/**
+ * Platform dependent utility methods
+ */
+
+package com.saveourtool.save.utils
+
+import com.saveourtool.save.core.logging.logDebug
+
+/**
```

Comment:
suggestion to add an interface:
```kotlin
interface EnvName {
    val name: String
}
```

and use it instead of `String` here and make `AgentEnvName` extends `EnvName` and introduce similar enum for demo